localization/english/wc_nicknames_l_english.yml

```diff
@@ -152,7 +164,6 @@
  nick_the_bloodletter:0 "the Bloodletter"
  nick_the_wretched:0 "the Wretched"
  nick_the_dead:0 "the Dead"
- nick_the_harmonious:0 "the Harmonious"
```

Comment:
Again why delete?
It's supposed to be used for [Zhi the Harmonious](https://warcraft.wiki.gg/wiki/Zhi_the_Harmonious)

Reply:
See previous answer - its a duplicate, someone added it to him while I had already given it to him. This one doesnt have a description though